src/test/java/seedu/progresschecker/testutil/TypicalTaskArgs.java

```diff
@@ -0,0 +1,43 @@
+package seedu.progresschecker.testutil;
+
+//@@author EdwardKSG
+/**
+ * A utility class containing a list of arguments to be used in tests for tasks commands.
+ */
+public class TypicalTaskArgs {
+    // week number arguments
+    public static final String FIRST_WEEK = "1";
+    public static final int FIRST_WEEK_INT = 1;
```

Comment:
You can actually use java method to convert String to int or vice versa when you need it in other java files such that you'll only have one constant variable for each of these doubles. That way, you can trim down the number of constant variables that you need to define. But I don't believe this is an issue; just a comment!